maltd-frontend/src/components/composite/UserSearch/UserSearch.test.js

```diff
@@ -43,18 +39,4 @@ describe("User Search", () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
-
-  test("Component renders with a loading spinner when in the loading state", () => {
```

Comment:
need a test to test this same functionality even without a spinner, but not a blocker for this PR